<<<<<<< HEAD
# Service to get joint angles of the robot

---
float64 j1
float64 j2
float64 j3
float64 j4
float64 j5
float64 j6
int64 ret
=======
# Service to get joint angles of the robot

---
float64 j1
float64 j2
float64 j3
float64 j4
float64 j5
float64 j6
int64 ret
>>>>>>> 1b1f87b6247333221ca1660d810eda01b82fd03f
string msg